NGitLab/GitLabException.cs

```diff
@@ -37,8 +37,6 @@ public GitLabException(string message, Exception inner)
         /// </summary>
         public HttpStatusCode StatusCode { get; set; }
 
-        public JsonObject ErrorObject { get; set; }
```

Comment:
I think you can preserve this property (but changing the return type) by using `JsonExtensionDataAttribute`

Reply:
I think it's a mistake to leak in our API any types that come from our current serializer. If later on we were to replace System.Text.Json by something else, we would need to modify our API once again.

Besides, has that property ever been useful? If so, I propose that we define it as something more generic, like `IDictionary<string, object>`.

Reply:
Done